### DISCUSSION TITLE
Error report in Docker installation.

### DISCUSSION THREAD

**Discussion by skyyxy** [pimcore/pimcore#15911] [Q&A / Support]
Hi everyone here, im the newguy for Pimcore and doing installation with docker under Ubuntu20.04 in VM. 
I follow the offical docker installation book, when I run: "docker compose exec php vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore " then show me:"ERROR: NO such service: php.

How I should todo? 
Thanks a lot.

**Comment by jdreesen** [comment]
Did you run `docker compose up -d` first to start the containers?
